### ISSUE TITLE
docs: broken references to https://mon.ooni.nu

### ISSUE THREAD

**Issue by ainghazal** [ooni/backend#607]
In several places (ex: README in https://github.com/ooni/pipeline) a pointer to a grafana instance can be found that seems not to resolve nowadays. Has this been moved somewhere else?

**Comment by hellais** [comment]
Good call, I think in general the pipeline docs (and a lot of the backend ones in general) could benefit from a bit of sprint cleaning.

These links should be pointing to https://grafana.ooni.org/, yet they are behind a login panel so you need credentials to access them.

**Comment by hellais** [comment]
We now have new docs. Future issues related to documentation should be filed against the related repo, see: https://docs.ooni.org/meta/
